docs/ffmpeg-params-guide.md

```diff
@@ -0,0 +1,187 @@
+| `interface_id` | Set the interface ID for the media data reception.       | Integer          | `0`                   |                                                  |
+
+## FFmpeg Media Communications Mesh Muxer Plugin Documentation
+
+The Mesh's Muxer plugin for FFmpeg is designed to handle the transmission of media data over a network using various protocols and payload types. Below are the input parameters that can be configured for the Mesh's Muxer plugin.
+
+### Input Parameters
+
+#### `ip_addr`
+- **Description**: Set the remote IP address to which the media data will be sent.
+- **Type**: String
+- **Default**: `"192.168.96.2"`
+- **Flags**: Encoding parameter
+
+#### `port`
+- **Description**: Set the remote port to which the media data will be sent.
+- **Type**: String
+- **Default**: `"9001"`
+- **Flags**: Encoding parameter
+
+#### `payload_type`
+- **Description**: Set the payload type for the media data.
+- **Type**: String
+- **Default**: `"st20"`
+- **Flags**: Encoding parameter
+- **Possible Values**: `"st20"`, `"st22"`, ~~`"st30"`~~, ~~`"st40"`~~, ~~`"rtsp"`~~
+
+#### `protocol_type`
+- **Description**: Set the protocol type for the media data transmission.
+- **Type**: String
+- **Default**: `"auto"`
+- **Flags**: Encoding parameter
+- **Possible Values**: `"memif"`, `"udp"`, `"tcp"`, ~~`"http"`~~, ~~`"grpc"`~~, `"auto"`
+
+#### `video_size`
+- **Description**: Set the video frame size using a string such as `"640x480"` or `"hd720"`.
+- **Type**: Image size (String)
+- **Default**: `"1920x1080"`
+- **Flags**: Encoding parameter
+
+#### `pixel_format`
+- **Description**: Set the video pixel format.
+- **Type**: Pixel format (Enum)
+- **Default**: `AV_PIX_FMT_YUV422P10LE`
+- **Flags**: Encoding parameter
+- **Possible Values**: ~~`AV_PIX_FMT_NV12`~~, `AV_PIX_FMT_YUV422P`, `AV_PIX_FMT_YUV444P10LE`, `AV_PIX_FMT_RGB24`, `AV_PIX_FMT_YUV422P10LE`
+
+#### `frame_rate`
+- **Description**: Set the video frame rate.
+- **Type**: Video rate (String or Rational)
+- **Default**: `"25"`
+- **Flags**: Encoding parameter
+
+#### `socket_name`
+- **Description**: Set the memif socket name for the media data transmission.
+- **Type**: String
+- **Default**: `NULL`
+- **Flags**: Encoding parameter
+
+#### `interface_id`
+- **Description**: Set the interface ID for the media data transmission.
+- **Type**: Integer
+- **Default**: `0`
+- **Flags**: Encoding parameter
+- **Range**: `-1` to `INT_MAX`
+
+### Example Usage
+
+To use the Mesh's Muxer plugin with FFmpeg, pass the `-f mcm` flag and parameters. Here is an example command that sets some of the parameters:
+
+```bash
+ffmpeg -i input.mp4 -c:v rawvideo -f mcm -ip_addr 192.168.1.100 -port 8000 -payload_type st20 -pixel_format yuv422p10le -protocol_type auto -video_size hd720 -frame_rate 30 -socket_name my_socket -interface_id 2 output.mcm
```

Comment:
@Mionsz What output do you expect to be written to `output.mcm`? I don't get it

Reply:
fixed.